.pre-commit-config.yaml

```diff
@@ -10,7 +10,6 @@ exclude: |
   ^base_rest_pydantic/|
   ^datamodel/|
   ^extendable/|
-  ^graphql_base/|
```

Comment:
Could you do graphql_demo in this same PR, or apply https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29 in #324 so we can test on runboat?

Reply:
Done!